app/services/paypal_service/api/billing_agreements.rb

```diff
@@ -114,5 +115,22 @@ def do_charge_commission(community_id, info, m_acc, admin_acc, payment)
       end
     end
 
+    def error_payment(cid, txid, payment)
```

Comment:
Needs better name, mark_payment_errored, record_payment_error, or something. Now you can get the impression that the job of this method is to cause a payment error but actually is just records an error already happened.